blazor/chart/chart-types/range-step-area.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Range Step Area in Blazor Charts component | Syncfusion
+description: Checkout and learn here all about Range Step Area Chart in Syncfusion Blazor Charts component and much more.
+platform: Blazor
+control: Chart
+documentation: ug
+---
+
+# Range Step Area in Blazor Charts Component
+
+## Range Step Area
+
+[Range Step Area Chart](https://www.syncfusion.com/blazor-components/blazor-charts/chart-types/range-area-chart) shows variation in data values over time and fills the high and low range areas accordingly. To render a [Range Step Area Chart](https://www.syncfusion.com/blazor-components/blazor-charts/chart-types/range-area-chart), set the series [Type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_Type) as [RangeStepArea](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeriesType.html#Syncfusion_Blazor_Charts_ChartSeriesType_SplineRangeArea).
```

Comment:
[Range Step Area Chart](https://www.syncfusion.com/blazor-components/blazor-charts/chart-types/range-area-chart) shows variation in data values over time and fills the high and low range areas accordingly with a step line.

Change like this.

Reply:
changed